docker-compose.yml

```diff
@@ -19,34 +19,74 @@ services:
     ports:
       - "4001:4001"
       - "4002:4002"
+    networks:
+      - als-mojaloop-net
+    depends_on:
+      - mysql-als
+      - central-ledger
     volumes:
-       - ./docker/account-lookup-service/default.json:/opt/account-lookup-service/config/default.json
+      - ./src:/opt/account-lookup-service/src
+      - ./docker/account-lookup-service/default.json:/opt/account-lookup-service/config/default.json
+      - ./docker/wait-for:/opt/wait-for
+    healthcheck:
+      test: ["CMD", "sh", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:4001/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
+
+  central-ledger:
+    image: mojaloop/central-ledger:latest
+    container_name: als_central-ledger
+    command:
+      - "sh"
+      - "-c"
+      - "sh /opt/wait-for/wait-for-central-ledger.sh && node src/api/index.js"
+    ports:
+      - "3001:3001"
+    volumes:
+       - ./docker/central-ledger/default.json:/opt/central-ledger/config/default.json
        - ./docker/wait-for:/opt/wait-for
     environment:
       - LOG_LEVEL=info
+      - CLEDG_DATABASE_URI=mysql://central_ledger:password@mysql-cl:3306/central_ledger
+      - CLEDG_SIDECAR__DISABLED=true
+      - CLEDG_MONGODB__DISABLED=true
     networks:
       - als-mojaloop-net
     depends_on:
-      - mysql
+      - mysql-cl
+      - kafka
+    healthcheck:
+      test: ["CMD", "sh", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:3001/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
+
+  kafka:
+    image: johnnypark/kafka-zookeeper:2.3.0
+    container_name: als_kafka
+    volumes:
+      - ./docker/kafka/:/opt/kafka_2.12-2.3.0/config/
+    ports:
+      - "2181:2181"
+      - "9092:9092"
+    environment:
+      - ZOO_LOG4J_PROP=WARN
+    networks:
+      - als-mojaloop-net
     healthcheck:
-      test: ["CMD", "bash", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:3001/health"]
+      test: ["CMD", "/opt/kafka_2.12-2.3.0/bin/kafka-broker-api-versions.sh", "--bootstrap-server", "kafka:29092"]
       timeout: 20s
       retries: 10
+      start_period: 40s
       interval: 30s
 
-  mysql:
+  mysql-als:
     image: mysql/mysql-server
-    container_name: als_mysql
-    # Disable logging as it is far too verbose for debugging locally
-    logging:
-      driver: none
-    # Map mysql port to 3307 on host machine to prevent conflict with central-ledger's mysql container.
-    # To run ALS with 'npm start' locally (i.e outside of docker) with mysql running in a docker container,
-    # please change 'DATABASE_URL' port to '3307' in config/default.json.
+    container_name: als_mysql_central_ledger
     ports:
-      - "3307:3306"
```

Comment:
I changed this, as I don't think there should be a case where we need to be running docker containers for different services at once. In order to not clash with the central-ledger mysql container (running from a different repository), simply run `docker stop <container_name>` or `docker-compose stop`

Reply:
Ok Lewis, I see your point. 
It seems more compact to have the services dependencies defined here.

Wouldn't central-ledger require other services like ml-api-adapter, simulator etc. as defined in central-ledger's `docker-compose.yml` (in central-ledger's repo)? If it does, then I think it's better we just launch central-ledger separately and then keep the port separation, unless I'm missing something.

Reply:
...on a second thought, I do think there is a merit to being able to pick up a service repo and launching it with all it's dependencies in one place rather than having to spin up another repo separately.

Reply:
So the reason for this is because the ALS depends on the `central-ledger` (see the `SWITCH_ENDPOINT` setting in `default.json`) so a proper integration test environment needs `central-ledger`, which needs a separate database and kafka as well. 

It's kind of a pain, but I think a necessary pain.

Reply:
Ok.